services/spar/src/Spar/Scim/User.hs

```diff
@@ -111,7 +131,8 @@ instance Scim.UserDB ST.SparTag Spar where
             | Scim.isUserSchema schema -> do
               x <- runMaybeT $ case attrName of
                 "username" -> scimFindUserByHandle mIdpConfig stiTeam val
-                "externalid" -> scimFindUserByEmail mIdpConfig stiTeam val
+                "externalid" -> scimFindUserByExternalId mIdpConfig stiTeam val
+                -- TODO: what if we want to search by email, but the email is stored in `emails`, not `externalId`?
```

Comment:
When the externalId is not a valid email `scimFindUserByExternalId` fails to find the user by email. It succeeds however when searched by the external id. Do we need to fix this in this PR?

Reply:
God no, please don't!  i was merely marking this as TODO to remind me to think about it again later.

Reply:
K, removed the TODO